FirstProject/assets/login.less

```diff
@@ -0,0 +1,73 @@
+//Login.less
+
+body {
+  background-color:#fafafa;
+}
+
+header {
+  background: url('../img/feedback-banner.jpg') center center no-repeat;
+  background-size: cover;
+  height: 600px;
+
+  .grid {
+    height: 100%;
+  }
+}
+
+.login-form {
+  background-color: #ffffff;
+  align-self: center;
+  text-align: center;
+  padding-bottom: 30px;
+
+  .login-form-header {
```

Comment:
If I have the same form in more pages probably I will reuse this class.(just if is the case)